proposals/fh_discord.md

```diff
@@ -0,0 +1,47 @@
+# Florida Hackers Discord Chat 
+
+## Target audience
+
+<!-- Who are you expecting to impact based on your proposal? -->
+Not only will the discord improve communication for FH Organizers with permission based rooms(The ability to create rooms made for specific eyes only based on roles assigned in discord, which an infinite amount of roles can be assigned to, where rooms can be seen by everyone or a select few depending on assigned role), it also allows a more open community of Florida Hackers to be active on discord from the rest of Florida.
```

Comment:
Can you please link to some documentation around rules so that those reading the proposal can have some references?